src/tools/illink/src/linker/Resources/Strings.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Mono.Linker.Resources {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]
```

Comment:
I don't think you wanted to include this file in your changes. If this file is out of date, that's for the owner of the il linker to submit an update.